playbook.md

```diff
@@ -309,7 +334,7 @@ By the end of this session, the full team should be confident that there is enou
 TODO
 
 #### Retrospectives
-At the end of every sprint, the delivery manager facilitates a [retrospective](https://www.gov.uk/service-manual/agile/running-retrospectives.html) where the team can discuss how the sprint went. These sessions are attended by all the people involved in delivering the sprint along with the client team. 
+At the end of every sprint, the delivery manager facilitates a [retrospective](https://www.gov.uk/service-manual/agile/running-retrospectives.html) where the team can discuss how the sprint went. These sessions are attended by all the people involved in delivering the sprint along with the client team.
```

Comment:
Magda this is appearing in this pull request as my editor removed a tiny bit of whitespace so this is nothing new that I'm trying to add 😃

That said there should be retrospectives at the end of every project's sprint, if this isn't the case definitely bring it up with a DM!